### ISSUE TITLE
eval loss keeps going up never goes down

### ISSUE THREAD

**Issue by jasondesante** [tensorflow/magenta#885]
Hi I was training a model of performance_rnn with about a gigabyte of training data, 200mb of eval data.  My problem is when training, tensorboard is showing that the eval loss has been going up and up and up, while the training loss has been going down.  

It was to my understanding that both eval and training loss should go down until a certain time that the model is trained then the eval loss should stop moving or go up.  For me the eval loss has only gone up.  

What went wrong?  Was it making the training data that something happened, or is it the model?  Is it a bug?  I need some help please.

**Comment by cghawthorne** [comment]
This could happen if your model is overfitting to your training data, which can happen for a variety of reasons including a training set that is too small, a training set that is from a different distribution than the test set, or a model that is too powerful.

I'd suggest training a model with more data or possibly just stopping training early.

**Comment by jasondesante** [comment]
I am training with a larger dataset (26mb of midi = 74mb notestream = 8gb performance_rnn training data)

I figured that would be a lot larger dataset because the last one was only 1 gb of performance_rnn training data.  

This time after 12 hours of training, the training set is still at loss 5.....basically the exact same spot it was at when it started.......

I have to be missing something here because I feel like I'm following all the instructions and all I get is problems and things breaking.  Every single time I've done this the loss has gone down, except for now.  



Not sure why you closed the issue, this is most definitely an open issue!!

**Comment by jasondesante** [comment]
Another thing is how could I stop training early if eval goes up from the very beginning that would suggest the model is the most trained when it is at 0, which suggests the entire thing is a waste of time or broken?  I seriously will understand things if they make sense.
